.github/workflows/build.yaml

```diff
@@ -13,6 +13,9 @@ on:
     types:
       - published
 
+env:
+  REGISTRY: ghcr.io
+
 jobs:
   lint:
     name: Lint code using eslint
```

Comment:
```suggestion
    name: Lint code using eslint
    permissions:
      contents: read
      packages: write
```

Reply:
no need for a separate GitHub token in secrets

Reply:
A seperate GitHub token is needed for running the `prune` actions. If this workflow runs with the standard one nothing will happen and all untagged images must be delete by hand.

Check this one here: https://github.community/t/trigger-on-package-registry-publish-not-working/189656/2